### PR TITLE
Hood bits

### PR DESCRIPTION
**Pull request by vichik123**
This includes just moving the hood back and forth until it fricking dies.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/hood/commands/bits/CheckHoodPressure.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    @Override
+    public void initialize() {
+        timer.reset();
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        if(timer.advanceIfElapsed(Constants.Hood.HOOD_PRESSURE_BIT_DELTA_TIME)) {
+            hood.toggle();
```
Comment (adamv17) [inline src/main/java/frc/robot/subsystems/hood/commands/bits/CheckHoodPressure.java:27]: Add some helpful prints and maybe like the intake number of runs

**Review by adamv17 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -84,6 +84,10 @@ public static TalonFXConfiguration getConfiguration() {
         public static final boolean IS_COMPENSATING_VOLTAGE = true;
     }
 
+    public static class Hood {
+        public static final double HOOD_PRESSURE_BIT_DELTA_TIME = 0.1;
```
Comment (barel13) [inline src/main/java/frc/robot/Constants.java]: units

File: src/main/java/frc/robot/subsystems/hood/Hood.java [hunk 3]
```diff
@@ -53,6 +53,10 @@ public void close() {
         angleChanger.set(true);
     }
 
+    public void toggle() {
```
Comment (barel13) [inline src/main/java/frc/robot/subsystems/hood/Hood.java:59]: I'm sorry, documentation

File: src/main/java/frc/robot/subsystems/hood/commands/bits/CheckHoodPressure.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import frc.robot.Constants;
+import frc.robot.subsystems.hood.Hood;
+
+public class CheckHoodPressure extends CommandBase {
+    private final Hood hood;
+    private final Timer timer = new Timer();
+    private int runCount = 0;
+
+    public CheckHoodPressure(Hood hood) {
+        this.hood = hood;
```
Comment (barel13) [inline src/main/java/frc/robot/subsystems/hood/commands/bits/CheckHoodPressure.java:14]: `addRequirements`

**Review by barel13 — CHANGES_REQUESTED** [review]

**Review by adamv17 — APPROVED** [review]

**Review by barel13 — APPROVED** [review]

**Pull request merged**